packages/webapp-server/Dockerfile

```diff
@@ -3,4 +3,4 @@ WORKDIR /app
 COPY package.json .
 RUN yarn
 COPY . .
-CMD ["yarn", "dev:watch"]
```

Comment:
why was watch removed from the DF?

Reply:
I just missed it. I had removed the ":watch" from the command in package.json but I did not rebuild the Docker image, so it looked like everything was working with Docker compose. When I tried to rebuild the image it broke.